### DISCUSSION TITLE
use metasploit framework with webui

### DISCUSSION THREAD

**Discussion by FunnyWolf** [rapid7/metasploit-framework#15693] [Show and tell]
MSF is the best penetration testing tool I have seen so far. It is open source and has rich functions. It has been updated all the time
However, as a command line tool, MSF is very difficult for newcomers to penetration testing
I have developed a graphical tool based on MSF, which can use most of the functions of MSF on Web pages, and use Python to call MSF API through msfrpc
I hope it can help more people to use MSF easily

https://github.com/FunnyWolf/Viper

**Comment by smcintyre-r7** [comment]
Wow that looks really cool. Upvoted.
